src/cmd/uds.go

```diff
@@ -287,3 +286,19 @@ func chooseBundle(args []string) string {
 
 	return path
 }
+
+func UnmarshalAndValidateConfig(configFile []byte, bundleCfg *types.BundleConfig) error {
+	// read relevant config into DeployOpts.Variables
+	// need to use goyaml because Viper doesn't preserve case: https://github.com/spf13/viper/issues/1014
+	err := goyaml.UnmarshalWithOptions(configFile, &bundleCfg.DeployOpts, goyaml.Strict())
+	if err != nil {
+		return err
+	}
+	// validate config options
+	for optionName := range bundleCfg.DeployOpts.Options {
+		if !types.IsValidConfigOption(optionName) {
```

Comment:
I think this is only used in the `cmd` pkg, can we move it there and make it private?